### DISCUSSION TITLE
Environment specific configuration value

### DISCUSSION THREAD

**Discussion by yerinu2019** [cue-lang/cue#1809] [General]
Is there a recommended way to specify environment specific value using Cue?
If Cue is targeting configuration language, simple way of specifying per-environment configuration value would be necessary. 
Google internal Piccolo language like environment specific configuration way (for Boq configuration) would be nice.

**Comment by mvdan** [comment]
Perhaps take a look at `cue help injection`, which I believe is what you're after.

**Comment by verdverm** [comment]
linking #411 (combinatorial configuration) which is a different angle on the question / answer
